### PR TITLE
add csscomb and scss-lint config

### PR DESCRIPTION
**Pull request by mikheevdmitry**
@vast попробовал прикрутить csscomb и добавил scsslint config. 
Но есть пока проблема с csscomb https://github.com/csscomb/csscomb.js/issues/210.
Есть небольшое решение, но оно не всегда срабатывает с @media https://gist.github.com/mikheevdmitry/c16eac91a720196c4d57
или можно добавить в node_modules https://gist.github.com/mikheevdmitry/47a887e803c2ae4c316c чтоб сразу по всем файлам из консоли пробежаться

### CODE REVIEW

**Comment by vast** [comment]
:+1: on adding scsslint with the same config we use in rails-base.
:-1: on adding csscomb as part of build process. csscomb have some issues like https://github.com/csscomb/csscomb.js/issues/210 or https://github.com/csscomb/csscomb.js/issues/368

For example, the last one makes impossible using csscomb with fallbacks rules:

```css
background: image-url('assurance-heatmap.png') center center / cover; // IE9 fallback
background: linear-gradient(to right, $charts-red, $charts-green 95%);
```

csscomb will fail the build in that case and after running it you'll get:

```css
background: image-url('assurance-heatmap.png') center center / cover; // IE9 fallback
background: linear-gradient(to right, $charts-red, $charts-green 95%);
```

With above said, I think it makes sense to put `.csscomb.json` in static-base, rails-base, but do _not_ include csscomb as a part of build process.

**Comment by mikheevdmitry** [comment]
@vast ok, i remove csscomb from build process. Can I merge that and add in rails-base project?

File: scss-lint.yml [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+linters:
+  NameFormat:
+    enabled: true
+    allow_leading_underscore: true
+    convention: "[^A-Z]"
+
+  StringQuotes:
+    style: double_quotes
+
+  PropertySortOrder:
```
Comment (vast) [inline scss-lint.yml]: let's not force any particular sort order here for the same reasons

Comment (mikheevdmitry) [inline scss-lint.yml]: ok, I removed it

File: bin/build [hunk 2]
```diff
@@ -2,5 +2,5 @@
 
 set -e
 
-bin/scss-lint source/assets/stylesheets
+bin/scss-lint --config scss-lint.yml
```
Comment (vast) [inline bin/build]: forgot source/assets/stylesheets

Comment (vast) [inline bin/build]: my bad, it works well with `scss_files` option in `scss-lint.yml`. That means that we don't need `source/assets/stylesheets` here: https://github.com/fs/static-base/pull/56/files#diff-a8da778d56fdb695e50daf67e50c0fbcR5

Comment (mikheevdmitry) [inline bin/build]: thx

Comment (mikheevdmitry) [inline bin/build]: fixed, thx

**Comment by vast** [comment]
:heart: :+1:

**Pull request merged**